capitalize.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'decorator'
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] `.upcase` makes the whole word upcase where you need to capitalize only the first letter, so I would suggest you use `.capitalize` instead